committee/committee.go

```diff
@@ -227,13 +230,15 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 		for {
 			select {
 			case <-ec.terminate:
-				ec.terminate <- true
```

Comment:
this line is intended  to stop another go routine. subscribeNewblock

Reply:
thx, improved in latest push